packages/components/src/Alert.js

```diff
@@ -1,20 +1,23 @@
 import React from 'react'
 import Box from './Box'
 
-export const Alert = React.forwardRef((props, ref) => (
-  <Box
-    ref={ref}
-    {...props}
-    __themeKey="alerts"
-    __css={{
-      display: 'flex',
-      alignItems: 'center',
-      px: 3,
-      py: 2,
-      fontWeight: 'bold',
-      color: 'white',
-      bg: 'primary',
-      borderRadius: 4,
-    }}
-  />
-))
+export const Alert = React.forwardRef(function Alert(props, ref) {
```

Comment:
For my understanding: is the named `function` syntax still necessary with the Emotion `label` property added below?

Reply:
Also, holy smokes! I don't think I realized the `label` property would work more generally in the `sx` prop – that's probably worth putting in a guide in the docs or mentioning somewhere 🤔

Reply:
> For my understanding: is the named `function` syntax still necessary with the Emotion `label` property added below?

Just for React devtools

Reply:
Does the `displayName` static property no longer work for React devtools?

Reply:
It does but just not when you’re using `forwardRef` https://reactjs.org/docs/forwarding-refs.html#displaying-a-custom-name-in-devtools

Reply:
Just thought I'd double check it, it does indeed work with `forwardRef`! Strange, I am sure that it wasn't working at the time, anyway, I'd probably stick with this approach as it's what's recommended and is more concise.